src/pages/payment-methods/_mobile-accordian-item.tsx

```diff
@@ -23,8 +24,11 @@ const Notes = styled.div`
     margin: 0 -16px;
     padding: 16px;
 `
-
-const MobileAccordianItem = ({ locale, pd }) => {
+type MobileAccordianItemProps = {
+    locale: object
```

Comment:
Can we find the shape of `locale` and type it instead of just declaring it as `object`

Reply:
okay